config/routes.rb

```diff
@@ -3,6 +3,8 @@
 
   resources :articles do
     resources :comments, only: [:create, :destroy]
+    resources :likes, only: [:create, :destroy]
+    delete '/unlike' => 'likes#destroy'
```

Comment:
I think you can use default route with method DELETE to /likes/:id - it looks good and "/unlike" is not something we really need to create readable to user